WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -420,13 +441,31 @@ private void initViewModel() {
             } else {
                 mFloatingActionButton.hide();
             }
+
+            if (mMySiteImprovementsFeatureConfig.isEnabled()) {
+                if (fabUiState.isFocusPointVisible()) {
+                    mHandler.postDelayed(mShowFabFocusPoint, 200);
+                } else if (!fabUiState.isFocusPointVisible()) {
+                    mHandler.removeCallbacks(mShowFabFocusPoint);
+                    mHandler.post(() -> {
+                        View focusPointView =
+                                findViewById(R.id.fab_container).findViewById(R.id.quick_start_focus_point);
+                        if (focusPointView != null) {
+                            ViewParent directParent = focusPointView.getParent();
+                            if (directParent instanceof ViewGroup) {
+                                ((ViewGroup) directParent).removeView(focusPointView);
+                            }
+                        }
```

Comment:
It seems that we could replace this with `QuickStartUtils::removeQuickStartFocusPoint`?

Reply:
I've previously used that method but the problem I had there is that it wasn't removing the correct focus point. For example when you clicked on "Publish post" and then on "Update site title", the site title focus point was removed because that method just finds any view in the parent with that ID and it was basically a race condition.